src/lib.rs

```diff
@@ -685,6 +685,35 @@ impl Uuid {
         Uuid { bytes: b }
     }
 
+    /// Creates a v4 Uuid from rng bytes (ref issue #170)
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// use uuid::Uuid;
+    /// use uuid::UuidBytes;
+    ///
+    /// let bytes:UuidBytes = [70, 235, 208, 238, 14, 109, 67, 201, 185, 13, 204, 195, 90, 145, 63, 62];
+    ///
+    /// let uuid = Uuid::from_random_bytes(bytes);
+    /// let uuid = uuid.hyphenated().to_string();
+    ///
+    /// let expected_uuid = String::from("46ebd0ee-0e6d-43c9-b90d-ccc35a913f3e");
+    ///
+    /// assert_eq!(expected_uuid, uuid);
+    /// ```
+    ///
+    #[cfg(feature = "v4")]
```

Comment:
There is no reason to feature-gate the method at all.

Reply:
We are only generating v4 uuids from this. So why have it on other features?

Reply:
Because there is no reason to have to bring in `rand` if one does not need it. For example in Servo we won't use `new_v4` at all (and thus the `rand` in `uuid`) because we need a better RNG.

Reply:
ok Cool.